deployment/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.3'
+services:
+  manage:
+    build: ./scripts/mgmt
+    volumes:
+      - ./scripts/mgmt:/usr/local/src/
+      - $HOME/.aws:/root/.aws:ro
+    environment:
+      - AWS_PROFILE=${AWS_PROFILE:-open-apparel-registry}
+    working_dir: /usr/local/src
+    entrypoint:
+      - python3
+      - manage.py
```

Comment:
`manage.py` in this case refers to the custom Boto script, not the `django-admin` script. If this naming scheme is confusing, I can change it.